src/schema/posts.ts

```diff
@@ -1749,6 +1769,57 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         true,
       );
     },
+    fetchSmartTitle: async (
+      _,
+      { id }: { id: string },
+      ctx: Context,
+    ): Promise<GQLPostSmartTitle> => {
+      const post: Pick<Post, 'title' | 'contentMeta'> = await queryReadReplica(
```

Comment:
I think the whole handler can be wrapped inside of this because you always query from RR? It will also avoid multiple connections to the DB?

Reply:
I do one write, but I guess I could wrap it all inside RR and still call write like normal?

Reply:
ah I see, no let's leave it as is then